src/test/regress/expected/pg17.out

```diff
@@ -1417,6 +1417,61 @@ DROP TABLE test_local_table_expr CASCADE;
 DROP TABLE test_distributed_table_expr CASCADE;
 DROP TABLE test_partitioned_expr CASCADE;
 -- End of Test for ALTER TABLE ... ALTER COLUMN ... SET EXPRESSION
+-- Test: Access Method Behavior for Partitioned Tables
+-- This test verifies the ability to specify and modify table access methods for partitioned tables
+-- using CREATE TABLE ... USING and ALTER TABLE ... SET ACCESS METHOD, including distributed tables.
+-- Step 1: Create a partitioned table with a specified access method
+CREATE TABLE test_partitioned_alter (id INT PRIMARY KEY, value TEXT)
+PARTITION BY RANGE (id)
+USING heap;
+-- Step 2: Create partitions for the partitioned table
+CREATE TABLE test_partition_1 PARTITION OF test_partitioned_alter
+  FOR VALUES FROM (0) TO (100);
+CREATE TABLE test_partition_2 PARTITION OF test_partitioned_alter
+  FOR VALUES FROM (100) TO (200);
+-- Step 3: Distribute the partitioned table
+SELECT create_distributed_table('test_partitioned_alter', 'id');
+ create_distributed_table
+---------------------------------------------------------------------
+
+(1 row)
+
+-- Step 4: Verify that the table and partitions are created and distributed correctly
```

Comment:
We have to verify step 4 in one of the worker nodes as well

Reply:
checks added